mscore/shortcut.cpp

```diff
@@ -3516,6 +3516,16 @@ Shortcut Shortcut::_sc[] = {
          Icons::Invalid_ICON,
          Qt::WindowShortcut
          },
+      {
+         MsWidget::SCORE_TAB,
+         STATE_NORMAL,
+         "realize-chord-symbols",
+         QT_TRANSLATE_NOOP("action","Realize Chord Symbols"),
+         QT_TRANSLATE_NOOP("action","Realize Chord Symbols"),
```

Comment:
2nd string could be `0` if identical to first, but here it actually should be slightly different, "Sentence case" rather than "Title Case", so `Realize chord symbols`

Reply:
Thank you, I'll change it!